cmd/tailtlsproxy/main.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
needs package doc.

But I'm not a huge fan of this proposed name.

I thought we were going to put this functionality in tailscaled.